DOL.WHD.Section14c.Web/src/modules/components/sectionWageData/wageDataPayTypeFormTemplate.html

```diff
@@ -1,50 +1,50 @@
 <div class="dol-form-question-group force-margin" ng-show="paytype">
   <div class="dol-form-question-block" ng-class="validate(modelPrefix() + '.numWorkers') ? 'usa-input-error' : ''">
     <div class="dol-form-question-help">
-      <label for="numWorkers" class="dol-form-question-text" id="numWorkersLabel">{{ paytype === 'hourly' ? 'How many workers with disabilities were paid an hourly subminimum wage during the most recently completed fiscal quarter?' : 'How many workers with disabilities received subminimum wages and were paid piece rates during the most recently completed fiscal quarter?' }}</label>
+      <label for="{{ paytype === 'hourly' ? 'h' : 'pr'}}NumWorkers" class="dol-form-question-text" id="{{ paytype === 'hourly' ? 'h' : 'pr'}}NumWorkersLabel">{{ paytype === 'hourly' ? 'How many workers with disabilities were paid an hourly subminimum wage during the most recently completed fiscal quarter?' : 'How many workers with disabilities received subminimum wages and were paid piece rates during the most recently completed fiscal quarter?' }}</label>
```

Comment:
What do you think about adding a function to capture this logic in one place?  Something like this:

```js
this.prefixedName = function(name) {
  return `${paytype === 'hourly' ? 'h' : 'pr'}${name}`;
}
```

and then use one-time binding in the template for performance reasons:

```html
<label for="::vm.prefixedName('NumWorkers')" class=...
```

My thinking is just that in other places, we've avoided putting a lot of logic in the templates.  This also makes it easier to change that logic in the future because it's all in one place.  I don't imagine it'll change, but 🤷‍♀️